native/src/main/java/io/ballerina/stdlib/sql/utils/ProcedureCallResultUtils.java

```diff
@@ -51,8 +51,8 @@
 /**
  * This class provides functionality for the `ProcedureCallResult` to iterate through the sql result sets.
  */
-public class ProcedureCallResultUtils {
-    public ProcedureCallResultUtils() {
+public final class ProcedureCallResultUtils {
```

Comment:
```suggestion
public class ProcedureCallResultUtils {
```

Reply:
These two classes are extended in other SQL connector. Hence removing the final keyword